3ds/WildMidi/PKGBUILD

```diff
@@ -0,0 +1,43 @@
+# Maintainer: WinterMute <davem@devkitpro.org>
+
```

Comment:
Add yourself in as contributor & I think we're good to merge this.

Reply:
Done. Thank you for being so helpfull.